example/lib/screens/login_screen.dart

```diff
@@ -75,13 +74,10 @@ class _LoginScreenState extends State<LoginScreen> {
     }
   }
 
-  preferredAuthentication() async {
+  preferredAuthentication(String profileId) async {
```

Comment:
Method name should contain a `verb` in order to describe what it actually does. So probably adding `get` would be fine.

Reply:
done